projects/ngx-formentry/src/components/ngx-datetime-picker/ngx-datetime-picker.component.ts

```diff
@@ -17,21 +17,19 @@ const moment = moment_;
 })
 export class NgxDatetimeComponent implements ControlValueAccessor {
   value: String | Date = '';
-  isDisabled: boolean;
+  isDisabled: Boolean = false;
```

Comment:
`boolean` => `Boolean` is the constructor

Reply:
(But since you initialize it anyway there is no need to be explicit about the type here)

Reply:
Should be sorted now